Neos.Flow/composer.json

```diff
@@ -54,12 +54,8 @@
     },
     "replace": {
         "typo3/flow": "self.version",
-        "symfony/polyfill-php54": "*",
-        "symfony/polyfill-php55": "*",
-        "symfony/polyfill-php56": "*",
         "symfony/polyfill-php70": "*",
         "symfony/polyfill-php71": "*",
-        "symfony/polyfill-php72": "*",
         "symfony/polyfill-mbstring": "*"
```

Comment:
replacing `symfony/polyfill-mbstring` without having a requirement for `ext-mbstring` looks also wrong to me

Reply:
Right… it should be replaced in Utility.Unicode, done.